test/azure/legacy/specification/packagemodecustomize/template/setup.py

```diff
@@ -12,7 +12,7 @@
 from setuptools import setup, find_packages
 
 
-PACKAGE_NAME = "{{ package_name }}"
+PACKAGE_NAME = "azure-packagemode-customize"
```

Comment:
this file should be `setup.py.jinja2` because it's formatted as a jinja template. Endings without `.jinja2` means "keep as is" inside the custom package-mode folder, so we should be able to handle both

Reply:
> this file should be `setup.py.jinja2` because it's formatted as a jinja template. Endings without `.jinja2` means "keep as is" inside the custom package-mode folder, so we should be able to handle both

sure

Reply:
some comments are same with up and they are resolved